docs/other-api/create-remix.md

```diff
@@ -24,14 +24,16 @@ npx create-remix@latest --help
 
 ### Package managers
 
-`create-remix` can also be invoked using the `create` command of various package managers, allowing you to choose between npm, Yarn and pnpm for managing the install process.
+`create-remix` can also be invoked using the `create` command of various package managers, allowing you to choose between npm, Yarn, pnpm, and Bun for managing the install process.
```

Comment:
Rewording since Bun doesn't provide the same `create` command as other package managers:

```suggestion
`create-remix` can also be invoked using various package managers, allowing you to choose between npm, Yarn, pnpm, and Bun for managing the install process.
```